policyengine_us/variables/gov/states/or/tax/income/credits/wfhdc/or_wfhdc_table_letter.py

```diff
@@ -0,0 +1,85 @@
+from policyengine_us.model_api import *
+
+
+class or_wfhdc_table_letter(Variable):
```

Comment:
similar to above - map to policy rather than visual tax form representation
```suggestion
class or_wfhdc_income_category(Variable):
```